vowpalwabbit/io/include/vw/io/logger.h

```diff
@@ -65,161 +65,161 @@ using logger_legacy_output_func_t = void (*)(void*, const std::string&);
 
 namespace details
 {
-const constexpr char* default_pattern = "%^[%l]%$ %v";
+const constexpr char* DEFAULT_PATTERN = "%^[%l]%$ %v";
 struct logger_impl
 {
-  std::unique_ptr<spdlog::logger> _spdlog_stdout_logger;
-  std::unique_ptr<spdlog::logger> _spdlog_stderr_logger;
-  size_t _max_limit = SIZE_MAX;
-  size_t _log_count = 0;
-  output_location _location = output_location::compat;
+  std::unique_ptr<spdlog::logger> spdlog_stdout_logger;
+  std::unique_ptr<spdlog::logger> spdlog_stderr_logger;
+  size_t max_limit = SIZE_MAX;
```

Comment:
Why remove the underscore here?

Reply:
they are public fields, only private fields should be prefixed with an underscore